test.rb

```diff
@@ -0,0 +1,19 @@
+a = 1
+  class AAA
```

Comment:
__[development][rubocop]__
* Useless assignment to variable - `a`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Inconsistent indentation detected.

<!-- Comment by [SideCI](https://sideci.com) -->